cmd/snapctl/flags.go

```diff
@@ -129,7 +125,7 @@ var (
 	flTaskMaxFailures = cli.IntFlag{
 		Name:  "max-failures",
 		Usage: "The number of consecutive failures before snap disable the task (Default 10 consective failures)",
-		Value: DefaultMaxFailures,
+		Value: -1,
```

Comment:
PR has been updated, @IRCody; here's the new help output for creating tasks:
```bash
$ snapctl task create --help
NAME:
   snapctl task create - There are two ways to create a task.
	1) Use a task manifest with [--task-manifest]
	2) Provide a workflow manifest and schedule details.

	* Note: Start and stop date/time are optional.


USAGE:
   snapctl task create [command options] [arguments...]

DESCRIPTION:
   Creates a new task in the snap scheduler

OPTIONS:
   --task-manifest value, -t value	File path for task manifest to use for task creation.
   --workflow-manifest value, -w value	File path for workflow manifest to use for task creation
   --interval value, -i value		Interval for the task schedule [ex (simple schedule): 250ms, 1s, 30m (cron schedule): "0 * * * * *"]
   --start-date value			Start date for the task schedule [defaults to today]
   --start-time value			Start time for the task schedule [defaults to now]
   --stop-date value			Stop date for the task schedule [defaults to today]
   --stop-time value			Start time for the task schedule [defaults to now]
   --name value, -n value		Optional requirement for giving task names
   --duration value, -d value		The amount of time to run the task [appends to start or creates a start time before a stop]
   --no-start				Do not start task on creation [normally started on creation]
   --deadline value			The deadline for the task to be killed after started if the task runs too long (All tasks default to 5s)
   --max-failures value			The number of consecutive failures before snap disable the task (default: -1)


$ 
```
Note that the reference to that default value has been removed.  Hopefully that addresses your concern here.